WooCommerce/WooCommerceTests/ViewRelated/Products/Media/ProductImageUploaderTests.swift

```diff
@@ -193,4 +195,199 @@ final class ProductImageUploaderTests: XCTestCase {
                                                                      isLocalID: true,
                                                                      originalStatuses: []).productImageStatuses)
     }
+
+    // MARK: - Status Updates
+
+    func test_update_is_emitted_when_image_upload_fails() {
```

Comment:
Perhaps `test_error_is_emitted_when_image_upload_fails`? And, can we follow the same for other test methods of this type as well?